uat-vocabularies/geo-commodities.ttl

```diff
@@ -3779,6 +3779,14 @@ cdt:greenhouse-gases a skos:Concept ;
         "Carbon Capture"@en ;
     skos:prefLabel "Greenhouse Gases"@en .
 
+cdt:quarry-product a skos:Concept ;
+    skos:broader cdt:sequestered-commodity ;
+    skos:definition "Any geocommodity produced by a quarry of economic value."@en ;
+    rdfs:isDefinedBy <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:inScheme <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:notation "QP" ;
+    skos:prefLabel "Quarry Product"@en .
+
```

Comment:
I think this is what is trying to be achieved. A collection for the webform to reference and generate a dropdown list for extractive (Quarry) resources. @DavidArmstrongGSQ ?

```suggestion
cdt:quarry-product a skos:Collection ;
    skos:definition "Any geocommodity produced by a quarry of economic value as applicable to Queensland statisical sales reporting."@en ;
    skos:member cdt:asphalt-aggregate,
        cdt:asphalt-sand,
        cdt:broken-rock,
        cdt:concrete-aggregate,
        cdt:pre-cast-concrete-sand,
        cdt:crushed-gravel,
        cdt:crusher-fines,
        cdt:decorative-crushed-rock,
        cdt:decorative-gravel,
        cdt:packing-sand,
        cdt:manufactured-sand,
        cdt:mortar-sand,
        cdt:other-coarse-aggregate,
        cdt:other-sand,
        cdt:ridge-gravel,
        cdt:plaster-sand,
        cdt:pre-mix-concrete-sand,
        cdt:rail-ballast,
        cdt:roadbase,
        cdt:sealing-aggregate,
        cdt:horticultural-sand,
        cdt:unprocessed-construction-material,
        cdt:unprocessed-gravel ;
    skos:prefLabel "Queensland Quarry Products"@en .

```

Reply:
Decided duplicate of and no need to add http://linked.data.gov.au/def/geo-commodities/qld-extractive-sales